src/modules/vtol_att_control/vtol_type.cpp

```diff
@@ -416,7 +416,7 @@ bool VtolType::set_motor_state(const motor_state target_state, const int32_t cha
 	case motor_state::DISABLED:
 		for (int i = 0; i < num_outputs_max; i++) {
 			if (is_channel_set(i, channel_bitmap)) {
-				_current_max_pwm_values.values[i] = _disarmed_pwm_values.values[i];
```

Comment:
Instead of introducing yet another parameter: How about setting the motors to PWM_DEFAULT_MIN (1000)? Or do you know of an ESC that starts spinning already at this value?

Reply:
I haven't seen an ESC start spinning at that value. I have seen ESCs disarm when the minimum is too low. Setting it to PWM_DEFAULT_MIN is probably fine. I'm probably being overly cautious by making it configurable. I'll set the maximum to PWM_DEFAULT_MIN.

Reply:
The point was that this parameter allows you to tailor your "in-air disarmed" PWM value to a value in the range that the manufacturer's datasheet specifies as being a valid value to keep the system initialized, but still below the lower end of the throttle range.  1000us is a little too close to the edge of some ESCs de-initialization limit for comfort.